src/pages/match/MatchSighting.jsx

```diff
@@ -109,6 +109,28 @@ export default function MatchSighting() {
     [matchResults, selectedQueryAnnotation],
   );
 
+  const confirmMatchHref = useMemo(
+    () => {
+      const individualGuid1 =
+        selectedQueryAnnotation?.individual_guid;
+      const individualGuid2 = selectedMatchCandidate?.individual_guid;
+      const encounterGuid1 = selectedQueryAnnotation?.encounter_guid;
+      const encounterGuid2 = selectedMatchCandidate?.encounter_guid;
+      if (individualGuid1 && individualGuid2) {
+        return `/merge?i=${individualGuid1}&i=${individualGuid2}`;
+      } else if (individualGuid1 || individualGuid2) {
+        const individualGuid = individualGuid1 || individualGuid2;
+        const encounterGuid = individualGuid1
+          ? encounterGuid2
+          : encounterGuid1;
+        return `/assign-annotations?i=${individualGuid}&e${encounterGuid}`;
```

Comment:
Missing = in e=

Reply:
Also, individualGuid1 or individualGuid2?

Reply:
see line 122 for that